tests/univariate/test_gaussian.py

```diff
@@ -0,0 +1,169 @@
+        assert copula.min == min_value
+        assert copula.max == max_value
+
+    def test_fit_empty_data(self):
+        """On fit, if column is empty an error is raised."""
+
+        # Setup
+        copula = GaussianUnivariate()
+        column = []
+
+        # Run
+        with self.assertRaises(ValueError):
+            copula.fit(column)
+
+    def test_test_fit_equal_values(self):
+        """On fit, even if column has equal values, std is never 0."""
+
+        # Setup
+        copula = GaussianUnivariate()
+        column = [1, 1, 1, 1, 1, 1]
+
+        # Run
+        copula.fit(column)
+
+        # Check
+        assert copula.mean == 1
+        assert copula.std == 0.001
+        assert copula.max == 1
+        assert copula.min == 1
+
+    def test_get_pdf(self):
+        """get_pdf returns the normal probability distribution value for the given values."""
+
+        # Setup
+        copula = GaussianUnivariate()
+        column = [-1, 0, 1]
+        copula.fit(column)
+        expected_result = 0.48860251190292
+
+        # Run
+        result = copula.get_pdf(0)
+
+        # Check
+        assert result == expected_result
+
+    def test_get_cdf(self):
+        """get_cdf returns the cumulative distribution function value for a point."""
+
+        # Setup
+        copula = GaussianUnivariate()
+        column = [-1, 0, 1]
+        copula.fit(column)
+        x = pd.Series([0])
+        expected_result = [0.5]
+
+        # Run
+        result = copula.get_cdf(x)
+
+        # Check
+        assert (result == expected_result).all()
+
+    def test_inverse_cdf(self):
+        """inverse_cdf returns the original point from the cumulative probability value """
+
+        # Setup
+        copula = GaussianUnivariate()
+        column = [-1, 0, 1]
+        copula.fit(column)
+        x = 0.5
+        expected_result = 0
+
+        # Run
+        result = copula.inverse_cdf(x)
+
+        # Check
+        assert (result == expected_result).all()
+
+    def test_inverse_cdf_reverse_get_cdf(self):
+        """Combined get_cdf and inverse_cdf is the identity function."""
+
+        # Setup
+        copula = GaussianUnivariate()
+        column = [-1, 0, 1]
+        copula.fit(column)
+        initial_value = pd.Series([0])
+
+        # Run
+        result_a = copula.inverse_cdf(copula.get_cdf(initial_value))
+        result_b = copula.get_cdf(copula.inverse_cdf(initial_value))
+
+        # Check
+        assert (initial_value == result_a).all()
+        assert (initial_value == result_b).all()
+
+    def test_sample(self):
+        """After fitting, GaussianUnivariate is able to sample new data."""
+        # Setup
+        copula = GaussianUnivariate()
+        column = [-1, 0, 1]
+        copula.fit(column)
+
+        # Run
+        result = copula.sample(1000000)
+
+        # Check
+        assert len(result) == 1000000
+        assert (np.mean(result) - copula.mean) < 10E-3
+        assert (np.std(result) - copula.std) < 10E-3
```

Comment:
I think it is better to use `self.assertAlmostEquals` or absolute value here since `np.mean(result)` could be much smaller than `copula.mean` but the assert above is still true.

Reply:
Right! I'll add the absoulte value.